azurelinuxagent/ga/exthandlers.py

```diff
@@ -1396,3 +1469,107 @@ def is_update_with_install(self):
         if update_mode is None:
             return True
         return update_mode.lower() == "updatewithinstall"
+
+
+class HandlerConfiguration(object):
```

Comment:
can we move this to its own file? thanks

Reply:
We can discuss this.

I can take up a task of refactoring the extHandlers.py to move the util classes we have to parse the files from extension package, and create helper files. Util classes such as - `HandlerEnvironment`, `HandlerManifest` and `HandlerConfiguration` (`CpuLimits`, `CpuLimit` and `MemoryLimit` are used in `HandlerConfiguration`).